src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 5; i++) {
+            System.out.println(lottery.getRandomColor() + "\n");
```

Comment:
```suggestion
            System.out.println(lottery.getRandomColor());
```
better use System.lineSeparator() than \n, however we don't need anything here - println prints a line + newline symbol